docpress.json

```diff
@@ -1,5 +1,9 @@
 {
   "docs": "documentation",
   "github": "fiverr/passable",
-  "dist": "docs"
-}
\ No newline at end of file
+  "dist": "docs",
+  "css": [
+      "https://unpkg.com/highlight.js@9.12.0/styles/atom-one-light.css",
+      "documentation/assets/style/main.css"
+  ]
```

Comment:
while at it, what do you think about adding passable directly as a script tag to the page, so people can use the console to play with it?

Reply:
I'll look into that 👍

Reply:
`docpress` doesn't behave nicely when trying to add local scripts. Maybe we can use the unpkg version in a future improvement.